specs/2.0.0/index.html

```diff
@@ -3858,6 +3858,16 @@ <h3><a name="NO_ETC2_EAC">No ETC2 and EAC compressed texture formats</a></h3>
            <code>WEBGL_compressed_texture_etc</code>.
     </div>
 
+    <h3><a name="MAYBE_DEPTH_COMPARE_FILTERED">E.g. <code>sampler2DShadow<code> does not guarantee e.g. <code>LINEAR<code> filtering</a></h3>
```

Comment:
This new text should be added to specs/latest/2.0/index.html rather than this file - possible to make that change?